src/util/graph.h

```diff
@@ -469,6 +471,37 @@ void grapht<N>::visit_reachable(node_indext src)
   }
 }
 
+template<class N>
+std::vector<typename N::node_indext>
+grapht<N>::get_reachable(node_indext src, bool forwards) const
```

Comment:
I'm not sure I like the fact that this basically does the same as `visit_reachable`, but just returns  the information in a different format -- yet neither is implemented in terms of the other.